src/roi/Roi.ts

```diff
@@ -13,9 +13,13 @@ import { RoiMap } from './RoiMapManager';
 import { getBorderPoints } from './getBorderPoints';
 import { getMask, GetMaskOptions } from './getMask';
 
+interface ID {
+  id: number;
+  length: number;
+}
 interface Computed {
   perimeter: number;
-  borderIDs: number[];
+  borderIDs: ID[];
```

Comment:
rename to `borders`

Please confirm that this is internal + external and add a comment in 'getBorders'